test/ode/ode_convergence_tests.jl

```diff
@@ -181,6 +181,16 @@ for i = 1:2
   sim19 = test_convergence(dts,prob,Hairer4())
   @test sim19.𝒪est[:final] ≈ 4 atol=testTol
 
+  sim20 = test_convergence(dts,prob,PaRK2p5(threading=true))
```

Comment:
put it with the ERK methods

Reply:
Should I also add complex tests?

Reply:
I don't think that's needed here? There's already complex tests?

Reply:
https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/5390222e6dd85e8558c23e0cbed264d6e919e097/test/complex_tests.jl#L14
These explicit methods do have separate complex tests